experimental/packages/otlp-transformer/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@opentelemetry/otlp-transformer",
+  "private": true,
+  "publishConfig": {
+    "access": "restricted"
+  },
+  "version": "0.27.0",
+  "description": "Transform OpenTelemetry SDK data into OTLP",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js",
+  "scripts": {
+    "compile": "tsc --build tsconfig.all.json",
+    "clean": "tsc --build --clean tsconfig.all.json",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "tdd": "npm run test -- --watch-extensions ts --watch",
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.test.ts'",
+    "test:browser": "nyc karma start --single-run",
+    "watch": "tsc --build -w tsconfig.all.json"
+  },
+  "keywords": [
+    "opentelemetry",
+    "nodejs",
+    "grpc",
+    "protobuf",
+    "otlp",
+    "tracing",
+    "metrics"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.js.map",
+    "build/src/**/*.d.ts",
+    "LICENSE",
+    "README.md"
+  ],
+  "devDependencies": {
+    "@opentelemetry/core": "1.0.1",
+    "@types/mocha": "8.2.3",
+    "@types/webpack-env": "1.16.3",
+    "codecov": "3.8.3",
+    "istanbul-instrumenter-loader": "3.0.1",
+    "karma": "6.3.14",
+    "karma-chrome-launcher": "3.1.0",
+    "karma-coverage-istanbul-reporter": "3.0.3",
+    "karma-mocha": "2.0.1",
+    "karma-spec-reporter": "0.0.32",
+    "karma-webpack": "4.0.2",
+    "mkdirp": "1.0.4",
+    "mocha": "7.2.0",
+    "nyc": "15.1.0",
+    "protobufjs": "6.11.2",
+    "rimraf": "3.0.2",
+    "ts-loader": "8.3.0",
+    "ts-mocha": "8.0.0",
+    "typescript": "4.4.4",
+    "webpack": "4.46.0"
+  },
+  "dependencies": {
+    "@opentelemetry/api": "1.0.3",
+    "@opentelemetry/api-metrics": "0.27.0",
+    "@opentelemetry/core": "1.0.1",
+    "@opentelemetry/resources": "1.0.1",
+    "@opentelemetry/sdk-metrics-base": "0.27.0",
+    "@opentelemetry/sdk-trace-base": "1.0.1"
```

Comment:
Modern typescript (don't remember which version) type checks dependencies so it will complain on the libcheck if a dependency is missing in an end user installation. Same answer as API above for peer/regular dependency.

Reply:
I think that the typescript checks should be covered by the peer dependency requirement. Adding this package as a regular dependency might result in the installation of 2 different versions in node_modules, where here we only care about the types.
Anyhow, this is a minor. If you prefer to keep it, feel free to resolve this thread